LabController/src/bkr/labcontroller/proxy.py

```diff
@@ -681,83 +742,77 @@ def postinstall_done(self, recipe_id=None):
         return self.hub.recipes.postinstall_done(recipe_id)
 
     def status_watchdog(self, task_id):
-        """ Ask the scheduler how many seconds are left on a watchdog for this task
-        """
+        """Ask the scheduler how many seconds are left on a watchdog for this task"""
         logger.debug("status_watchdog %s", task_id)
         return self.hub.recipes.tasks.watchdog(task_id)
 
-    def task_stop(self,
-                  task_id,
-                  stop_type,
-                  msg=None):
-        """ tell the scheduler that we are stoping a task
-            stop_type = ['stop', 'abort', 'cancel']
-            msg to record if issuing Abort or Cancel """
+    def task_stop(self, task_id, stop_type, msg=None):
+        """tell the scheduler that we are stoping a task
```

Comment:
Don't know if you want to fix the typo s/b 'stopping'?  Up to you.

Reply:
I'll fix it.